edb/edgeql-parser/tests/tokenizer.rs

```diff
@@ -634,6 +634,8 @@ fn strings() {
     assert_eq!(tok_typ(" b\"hel\nlo\" "), [BinStr]);
     assert_eq!(tok_str(" b'hel\nlo' "), ["b'hel\nlo'"]);
     assert_eq!(tok_typ(" b'hel\nlo' "), [BinStr]);
+    assert_eq!(tok_str(" rb'hel\nlo' "), ["rb'hel\nlo'"]);
+    assert_eq!(tok_str(" br'hel\nlo' "), ["br'hel\nlo'"]);
```

Comment:
More tests needed:
1. Add `tok_typ` assertion, that makes sure that this is a binary string.
2. Add test that ensures that something that is not allowed in binary string like `\xFF` is allowed in raw string
3. A tests that disallows escaping quotes `rb'test\'`
4. Different quotes

Maybe look at other binary and raw string cases. This is quite sensitive piece of code. That we want all combinations of things tested against, in case we further optimize some tokenizer code paths in future. (Tiny exception I'm okay with: don't have to copy all the tests with `rb` and `br`, that seems to be trivially proven to work).

Reply:
Ah, yes, I misread `c if c as u32 > 0x7f` as if it doesn't allow hex escapes > 0x7f, while it doesn't allow actual character codes like that.